src/templates/weekEnded/html.mjml

```diff
@@ -1,41 +1,85 @@
-          <a href="{{ browserLink }}" class="link">View online version</a>
+
+  <mj-section padding="0px 0px 15px 0px">
+      <mj-column vertical-align="middle">
+        <mj-text align="left" font-size="15px" padding="0px">
+          <p style="text-align: center; margin: 10px 0px;">
+            <span>Email not displaying correctly? </span>
+            <a style="font-size: 15px;" href="{{ browserLink }}" class="link">View it in your browser</a>
+          </p>
+        </mj-text>
+      </mj-column>
+    </mj-section>
+
+    <mj-section background-repeat="no-repeat" background-size="100% 100%" background-url="{{ domain }}/bkgd-pitch.png" padding="25px 200px" border-radius="10px 10px 0px 0px" >
+      <mj-column background-color="transparent">
+        <mj-image align="center" alt="Football Image" border="none" height="auto" src="{{ domain }}/football.svg" title="Football Image" width="150px"></mj-image>
+      </mj-column>
+      </mj-section>
+
+       <mj-section padding="0px" text-align="center">
+      <mj-column background-color="#ffffff" width="100%">
+        <mj-text padding="15px 0px 0px 0px">
+          <h1 align="center">
+            <span style="letter-spacing:.5px;">
+              <strong>NFL Confidence Pool</strong>
+            </span>
+          </h1>
         </mj-text>
       </mj-column>
+
       <mj-column width="100%" background-color="#fff" border-radius="10px">
-        <mj-text padding="30px 0 15px 0" font-size="18px" align="center">
-          <h1>Hello {{ user.userFirstName }},</h1>
+        <mj-text padding="10px 0px 10px 0px" font-size="18px" align="center">
+          <p>Hello {{ userFirstName }},</p>
+          <p>Week {{ week }} has just ended with
+{{ visitorTeam }} @ {{ homeTeam }}.</p>
+          <p>The final score was {{ gameVisitorScore }}-{{ gameHomeScore }}.</p>
+         </mj-text>
+<mj-button align="center" background-color="#15803d" href="{{ domain }}" border-radius="3px" border="0px solid #ffffff" color="#ffffff" font-weight="normal" inner-padding="13px 25px 13px 25px" padding="10px 50px 10px 50px" text-decoration="none" text-transform="none" vertical-align="middle" width="100%">
+          <span style="font-size:16px;">Check Your Ranking</span>
+        </mj-button>
+        <mj-text align="center" color="#000000" padding="30px 25px 20px 25px">
+          <span>If you did not request this email you can safely ignore it.</span>
+        </mj-text>
+      </mj-column>
 
-          <p>Week {{ week }} has just ended with {{ visitorTeam.teamShortName }} @ {{ homeTeam.teamShortName }}.</p>
+    </mj-section>
 
-          <p>The final score is {{ game.gameVisitorScore }}-{{ game.gameHomeScore }}.</p>
-          <mj-text padding="20px 0 25px 0" font-size="16px" line-height="22px" align="center">
-            <a href="{{ domain }}/users/edit">Edit Notification Preferences</a>
-          </mj-text>
+    <mj-section background-size="auto" padding="50px 0px 20px 0px" text-align="center">
```

Comment:
@balexander82 If we're going to have things in every single email (such as this footer) that's where we can use partials to bring them in.  Let's chat about this tonight